### PR TITLE
Weather provider weatherflow

### PR DESCRIPTION
**Pull request by 10bias**
Hi everyone,

i added a weather provider for weatherflow (https://weatherflow.com/)

best regards

### CODE REVIEW

**Comment by khassel** [comment]
Thank you for the contribution. 

An additional PR in the [documentation repo](https://github.com/MichMich/MagicMirror-Documentation) would be helpful, a new section in this [document](https://github.com/MichMich/MagicMirror-Documentation/blob/master/modules/weather.md) for the new provider.

File: modules/default/weather/providers/weatherflow.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/* global WeatherProvider, WeatherObject */
+
+/* MagicMirror²
+ * Module: Weather
+ * Provider: Dark Sky
```
Comment (rejas) [inline modules/default/weather/providers/weatherflow.js:5]: wrong name :-)

**Comment by rejas** [comment]
Also you probably need to do a "npm run lint:prettier" and push the changes so that the linter wont complain once @MichMich approves the workflows

**Review by rejas — APPROVED** [review]

**Comment by 10bias** [comment]
Thanks for the hints. I fixed the copy and paste mistake in the comment and run the lint. I also added a documentation for the provider und created a pr: https://github.com/MichMich/MagicMirror-Documentation/pull/109

**Comment by 10bias** [comment]
I have just add a fix with the sunrise.

File: modules/default/weather/providers/weatherflow.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	providerName: "WeatherFlow",
+
+	// Set the default config properties that is specific to this provider
+	defaults: {
+		apiBase: "https://swd.weatherflow.com/swd/rest/",
+		token: "",
+		stationid: ""
+	},
+
+	units: {
```
Comment (rejas) [inline modules/default/weather/providers/weatherflow.js:26]: those seem unused?

**Comment by 10bias** [comment]
@rejas oh yes, i have added a functionality to switch between the units. I hope the imperial units a correct. Some researches didn't deliverd a clear result. If there is something wrong, please let me know.

**Comment by rejas** [comment]
Thx for the info. Cant tell if yomethings wrong, got no api keys :-) Since this a fresh provider, I dont mind merging it and see if it works "in the wild" :-)

**Review by rejas — APPROVED** [review]

**Comment by MichMich** [comment]
Thanks for your contribution! My apologies for my slow response. Life got in the way.

**Pull request merged**